src/CoreEditor/mentions/mention.js

```diff
@@ -16,6 +16,8 @@ class Mention {
 const fixUrlProtocol = url => {
   if (!url) {
     return;
+  } else if (location.origin === 'file://') {
```

Comment:
This code is part changes from commit about support pages rendering offline created by "Save As" button. This code doesn't break anything. We can leave it at this PR.